.prettierignore

```diff
@@ -1,4 +1,4 @@
 src/contracts
 src/ecsScene
 src/components/Preview/loader.json
-src/modules/analytics/rollbar.ts
+src/modules/analytics/sentry.ts
```

Comment:
What do you think about removing this line? This seems to be there because `rollbar.ts` included the snippet on its code, but now there's a configuration directive.